project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -2563,29 +2566,28 @@ function c4m_og_edit_author_autocomplete($string = '') {
     return;
   }
 
-  $gid = c4m_og_current_group_id();
-  // Return early if there is no group context.
+  // Resolve group id from auto complte path, which is set to
+  // 'edit-author/autocomplete/<$gid>'.
+  $destination = drupal_get_destination();
```

Comment:
Get the group id.
Previous implementation was relying on og_context(), which always returned NULL.